### DISCUSSION TITLE
Song playback randomly pauses when switching/restarting beatmaps

### DISCUSSION THREAD

**Discussion by GiantNuker** [ppy/osu#18297] [Ideas]
### Type

Game behaviour

### Bug description

Audio playback randomly pauses, usually happens when switching songs, though its quite rare that it happens at all.

- The song will freeze and resume some time later during the song, the hitsounds work fine. (it appears to be looping the pause sound in the video, but it also happens with no sound if you start a song that isnt playing from the beatmap menu)
- This bug first appeared some time this year (or I didn't see it before at least), i just haven't been able to record it fast enough / with audio before. (audio resume time is not related to actually playing, and can happen after pause/resume)
- A related issue might be: rarely when a song is restarted, it will start playing in the loading screen, then cut off and restart when the beatmap starts
- This happens independently of the beatmap (does not only affect the one in the video)
- This is not easy to reproduce (happens once in ~8 hours of runtime, completely randomly)

System Info:
- Arch linux (updated some time in the last month)
- **Pipewire**
- Wayland

### Screenshots or videos

https://streamable.com/qihh07

### Version

2022.509.0

### Logs

[database.log](https://github.com/ppy/osu/files/8695880/database.log)
[network.log](https://github.com/ppy/osu/files/8695881/network.log)
[performance.log](https://github.com/ppy/osu/files/8695882/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8695884/runtime.log)

**Comment by LOKSTED** [comment]
Looks like you should open an issue https://github.com/ppy/osu/issues

**Reply by bdach** [reply]
no, this should remain a discussion, because it is unclear what is causing this, and i posit this is likely to be related to the user's audio setup (i.e. pipewire). i for one cannot reproduce.
